bin/GameIndex.dbf

```diff
@@ -2281,6 +2281,12 @@ Serial = SCES-53680
 Name   = WRC avec Sebastien Loeb
 Region = PAL-M5
 XgKickHack = 1 // SPS.
+[patches = 79BAD675]
+	author=By Prafull and atomic83github
```

Comment:
It will be better to have Prafull as author only since there aren't any actual modifications made to the patch.